File: corgi-spark/src/main/scala/com/dyingbleed/corgi/spark/measure/Measure.scala
package com.dyingbleed.corgi.spark.measure

import org.joda.time.LocalDateTime

/**
  * Created by 李震 on 2018/6/13.
  */
case class Measure(
                    name: String,
                    submissionTime: LocalDateTime, // 提交时间
                    completionTime: LocalDateTime, // 完成时间
                    elapsedSeconds: Long, // 运行秒数
                    inputRows: Long, // 输入数据行数
                    inputData: Long, // 输入数据大小
                    outputRows: Long, // 输出数据行数
                    outputData: Long // 输出数据大小
                  )

File: corgi-web/src/main/java/com/dyingbleed/corgi/web/bean/Measure.java
package com.dyingbleed.corgi.web.bean;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;
import org.springframework.format.annotation.DateTimeFormat;

import java.util.Date;

/**
 * Created by 李震 on 2018/6/13.
 */
@Data
public class Measure {

    private Long id;

    private String name;

    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private Date submissionTime;

    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private Date completionTime;

    private Long elapsedSeconds;

    private Long inputRows;

    private Long inputData;

    private Long outputRows;

    private Long outputData;

}
